packages/epo-react-lib/vite.config.ts

```diff
@@ -25,6 +25,7 @@ export default defineConfig({
         "react-i18next",
       ],
       output: {
+        interop: "auto",
```

Comment:
learned that there is an [issue with styled-components in CommonJS libs](https://github.com/styled-components/styled-components/issues/3601), since Jest defaults to using the UMD lib it exposed this issue and required an update to the Vite Rollup config.